File: Linux/FileIO/fileio/fileio.cpp
#include <fcntl.h>
#include <unistd.h>
#include <iostream>

int main()
{
	int fd = open("text.txt", O_RDWR | O_APPEND);
	if (fd < 0)
		std::cout << "error" << std::endl;
	lseek(fd, 0, SEEK_CUR);
	int nbytes = write(fd, "Hello World!\n", 13);
	close(fd);
	std::cout << nbytes << std::endl;
}

File: ProgrammingLanguage/Python/FileIO/fileio.py
#!/usr/bin/python

fp = open('./test.txt', 'w')
fp.write('Hello world!\n')
fp.write('Hello yoleimei!\n')
fp.close()

fp = open('./test.txt', 'r')
ret = fp.readline()
print(ret)

fp = open('./test.txt', 'r')
ret = fp.readlines()
for i in ret:
	print(i)
